src/Models/Order.php

```diff
@@ -409,14 +409,17 @@ public function registerMediaCollections(): void
     {
         $this->addMediaCollection('invoice')
             ->acceptsMimeTypes(['application/pdf', 'image/jpeg', 'image/png', 'application/xml', 'text/xml'])
-            ->singleFile();
+            ->singleFile()
+            ->readOnly();
 
         $this->addMediaCollection('payment-reminders')
-            ->acceptsMimeTypes(['application/pdf']);
+            ->acceptsMimeTypes(['application/pdf'])
+            ->readOnly();
 
         $this->addMediaCollection('signature')
             ->acceptsMimeTypes(['image/jpeg', 'image/png'])
-            ->useDisk('local');
+            ->useDisk('local')
```

Comment:
why does this use local disk?

Reply:
local disk is not public accessible through a simple route.
All "sensitive" files should use local in my understanding